premake5.lua

```diff
@@ -27,6 +27,8 @@
 		trigger = "embed",
 		description = "Embed scripts in scripts.c; required before release builds",
 		execute = function ()
+			_MAIN_SCRIPT_DIR = os.getcwd()
+			_SCRIPT_DIR = path.join(_MAIN_SCRIPT_DIR, "scripts")
```

Comment:
Before this change, none of my lua scripts would get added to the embed... these two lines are in the premake4.lua script, but not in the premake5.lua script, so I copied them... made it work.